operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [BUGFIX] Fix an issue that caused metamonitoring secrets to be created incorrectly #3170
 * [BUGFIX] Nginx: fixed `imagePullSecret` value reference inconsistency. #3208
 * [BUGFIX] Move the activity tracker log from /data to /active-query-tracker to remove ignore log messages. #3169
+* [BUGFIX] Fix Invalid ingress nginx config due to newline in prometheusHttpPrefix Helm named templates. #3088
```

Comment:
should this be #3087 ?

Reply:
Yes. Thanks for noticing it!

Reply:
Thank you for catching this :(. I might accidentally increment the number during editing. Will be more careful next time.